classes/Pods.php

```diff
@@ -1109,27 +1109,16 @@ public function field( $name, $single = null, $raw = false ) {
 							} else {
 								// Fallback to default attachment object.
 								$attachment = get_post( $attachment_id );
-								$value      = pods_v( implode( '.', $traverse_names ), $attachment, '' );
-								if ( ! $value ) {
-									$meta_key = array_shift( $traverse_names );
-									$value    = get_post_meta( $attachment_id, $meta_key, true );
-
-									// Maybe traverse.
-									if ( count( $traverse_names ) ) {
-										if ( is_array( $value ) ) {
-											foreach ( $traverse_names as $field ) {
-												if ( ! isset( $value[ $field ] ) ) {
-													$value = null;
-
-													break;
-												}
-
-												$value = $value[ $field ];
-											}
-										} else {
-											$value = null;
-										}
+								$value      = pods_v( implode( '.', $traverse_names ), $attachment, null );
+
+								if ( null === $value ) {
+									// Start traversal though object property or metadata.
+									$name_key = array_shift( $traverse_names );
+									$value    = pods_v( $name_key, $attachment, null );
+									if ( null === $value ) {
+										$value    = get_post_meta( $attachment_id, $name_key, true );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 4 spaces

<sub>hash: 496321c1fdfbc85b36973cdecbce3f85</sub>